CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc

```diff
@@ -204,12 +204,29 @@ CaloTPGTranscoderULUT::getOutputLUTSize(const HcalTrigTowerDetId& id) const
       case HcalTopologyMode::TriggerMode_2016:
          return QIE8_OUTPUT_LUT_SIZE;
       case HcalTopologyMode::TriggerMode_2017:
+         if (id.ietaAbs() <= theTopology->lastHERing())
+            return QIE8_OUTPUT_LUT_SIZE;
+         else
+            return QIE10_OUTPUT_LUT_SIZE;
+      case HcalTopologyMode::TriggerMode_2017plan1:
+         if (theTopology->dddConstants()->isPlan1(id))
+            return QIE11_OUTPUT_LUT_SIZE;
```

Comment:
Doesn't it get filled [here](https://github.com/cms-sw/cmssw/blob/a85956d574ec10a14b2dba8a1d503d4ddd0bb208/Geometry/HcalCommonData/src/HcalDDDRecConstants.cc#L896)?